packages/react/src/components/Auth/Auth.tsx

```diff
@@ -165,7 +165,11 @@ function Auth({
     case VIEWS.SIGN_IN:
       return (
         <Container>
-          <EmailAuth {...emailProp} authView={'sign_in'} />
+          <EmailAuth
+            {...emailProp}
+            appearance={appearance}
```

Comment:
This should be added to the `emailProp` constant instead as that prop is spread on the component itself. I can see appearance is a part of the interface just not the implementation of the constant. I think the constant is on line 147.